pynetbox/core/endpoint.py

```diff
@@ -53,8 +84,12 @@ def __init__(self, api, app, name, model=None):
             endpoint=self.name,
         )
         self._choices = None
+        self._init_cache()
```

Comment:
This is only called in one place and has a function defined for it - can you just put it inline with a comment instead, not sure why we need a separate function for this?